src/Monolog/Handler/Handler.php

```diff
@@ -44,19 +44,27 @@ public function __destruct()
         }
     }
 
-    public function __sleep()
+    public function __serialize(): array
     {
         $this->close();
 
         $reflClass = new \ReflectionClass($this);
 
-        $keys = [];
+        $data = [];
         foreach ($reflClass->getProperties() as $reflProp) {
             if (!$reflProp->isStatic()) {
-                $keys[] = $reflProp->getName();
+                $reflProp->setAccessible(true);
```

Comment:
this call is useless. Monolog requires PHP 8.1+, and this call is a no-op in PHP 8 (and btw, it is *also* deprecated in PHP 8.5)